src/analyze/analyze-security.c

```diff
@@ -1922,27 +1924,21 @@ static int acquire_security_info(sd_bus *bus, const char *name, struct security_
         if (FLAGS_SET(flags, ANALYZE_SECURITY_ONLY_LONG_RUNNING) && streq_ptr(info->type, "oneshot"))
                 return -EMEDIUMTYPE;
 
-        if (info->private_devices ||
-            info->private_tmp ||
-            info->protect_control_groups ||
-            info->protect_kernel_tunables ||
-            info->protect_kernel_modules ||
-            !streq_ptr(info->protect_home, "no") ||
-            !streq_ptr(info->protect_system, "no") ||
-            info->root_image)
+        if (info->private_devices || info->private_tmp || info->protect_control_groups ||
+            info->protect_kernel_tunables || info->protect_kernel_modules ||
+            !streq_ptr(info->protect_home, "no") || !streq_ptr(info->protect_system, "no") || info->root_image)
                 info->private_mounts = true;
 
         if (info->protect_kernel_modules)
                 info->capability_bounding_set &= ~(UINT64_C(1) << CAP_SYS_MODULE);
 
         if (info->private_devices)
-                info->capability_bounding_set &= ~((UINT64_C(1) << CAP_MKNOD) |
-                                                   (UINT64_C(1) << CAP_SYS_RAWIO));
+                info->capability_bounding_set &= ~((UINT64_C(1) << CAP_MKNOD) | (UINT64_C(1) << CAP_SYS_RAWIO));
```

Comment:
this could be added to the formatter, but i am not sure if this is a good idea.
Its a little more complicated than a simple bit set, since the expression is negated. 
IMHO: When aligned on new line this gets easier confused  "~a | b" vs "~(a | b)"